src/AssetPublisher.php

```diff
@@ -59,13 +177,23 @@ public function getAssetUrl(AssetManager $am, AssetBundle $bundle, string $pathA
             throw new InvalidConfigException("Asset files not found: '$basePath/$pathAsset.'");
         }
 
-        if ($am->getAppendTimestamp()  && ($timestamp = @filemtime("$basePath/$pathAsset")) > 0) {
+        if ($this->appendTimestamp  && ($timestamp = @filemtime("$basePath/$pathAsset")) > 0) {
             return "$baseUrl/$pathAsset?v=$timestamp";
```

Comment:
Seems `$baseUrl` and `$basePath` might be undefined for asset bundle with CDN = false.

Reply:
No, check here https://github.com/terabytesoftw/assets/blob/73c37cd03efd280a5aef7819007e0f4092c496cd/src/AssetPublisher.php#L219

Reply:
https://github.com/terabytesoftw/assets/blob/73c37cd03efd280a5aef7819007e0f4092c496cd/src/AssetManager.php#L90

https://github.com/terabytesoftw/assets/blob/73c37cd03efd280a5aef7819007e0f4092c496cd/src/AssetPublisher.php#L219 check here before.

Reply:
The method is public so we cannot assume input is correct.

Reply:
Also the check won't work if CDN = true.

Reply:
done.